base/applications/network/netstat/netstat.c

```diff
@@ -119,16 +124,17 @@ BOOL ParseCmdline(int argc, wchar_t* argv[])
                     case L's':
                         bDoShowProtoStats = TRUE;
                         break;
-                    case L'o':
-                        bDoShowProcessId = TRUE;
+                    case L't':
+                        // UNIMPLEMENTED.
+                        ConPuts(StdErr, L"'t' option is FIXME (Accepted option though unimplemented feature).\n");
```

Comment:
Why?

Reply:
(Why what?) Added to match other FIXME options...